src/pages/favorites/Favorites.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { BsArrowRight } from "react-icons/bs";
+import { FiChevronLeft } from "react-icons/fi";
+import { HiOutlineMinusSm } from "react-icons/hi";
+import { Link } from "react-router-dom";
+import { useFavorites } from "../../stogre/addToLikes";
+import "./Favorites.css";
+const Favorites = () => {
+  const favoritesContext = useFavorites();
+  const favoriteProduct = favoritesContext.favorites;
+  const RemoveFavoriteHandler = (id) => {
+    const isFavorite = favoritesContext.itemIsFavorite(id);
+    if (isFavorite) {
+      return favoritesContext.removeFavorite(id);
```

Comment:
for these usages, you could use object restructuring and read them on hook usage like this:
` const { removeFavorite, itemIsFavorite } = useFavorites();`
and here it would be so simpler to use.